.github/ISSUE_TEMPLATE/language_bindings_request.pkl

```diff
@@ -0,0 +1,69 @@
+name = "Language Bindings Request"
```

Comment:
It's fine to postpone, because you might be able to iterate faster without, but adding a template for this (and start this with `amends "path/to/template/maybe/even/package/on/pkl-pantry"`) would give you the full Pkl experience ;)

You can also start by just putting type annotations in _this_ file.

Reply:
I was thinking about doing something like that but then I decided to just write the `yml` assisted by the existing template and basing the `pkl` on that, since it's not gonna be changed a lot and I didn't wanna bury myself in a rabbithole. 
I'm not gonna go that deep atm but I bet as this gets more popular someone is gonna make a template that can then be used here.
Really seeing the use-cases of this language tho :D

Reply:
I will write a template tho, where should I put that? I don't think it makes sense to put it in the ISSUE_TEMPLATE folder or one might confuse it with an actual issue template that didn't get rendered by accident. What do you think?

Reply:
In a very general sense; you can publish packages anywhere you like (that's just FYI for whatever else you might be looking to create templates for). All repositories maintained by the core Pkl team should really only be use packages in those repositories. This could be an excellent contribution to [pkl-pantry](https://github.com/apple/pkl-pantry). It's worth having a think about naming. I'm hesitant to suggest `com.github` as a package name, because that would suggest an authoritative source, but maybe `com.github.contrib`? (cc @bioball).

Reply:
I was gonna write just a template for this specific usecase, not a completely generalized one. This would just include the "GENERATED" message and shared attributes between similar templates. Obviously that's not really necessary, but I wanted to try out some of the features of pkl, without writing a complete implementation of github's issue form standard.

Reply:
That’s entirely fair. In that case, I’d keep it local and remove the `amends` before commit or - probably better - just do that type work inline (I.e. not a template, but proper validation)

Reply:
> I’d keep it local and remove the `amends` before commit

@holzensp what did you mean by this?

Reply:
> "this isn't a general implementation of the GitHub Issues API, it's just something used here."

Exactly. If there is enough demand, someone is gonna make a full suite of GitHub scoped templates, I'll leave that to them and use a simple template for this use case.

Reply:
So my question remains, where should I put the template file?